src/cpu/pred/bpred_unit.cc

```diff
@@ -526,7 +521,7 @@ BPredUnit::squash(const InstSeqNum &squashed_sn,
         set(hist->target,  corr_target);
 
         // Correct Direction predictor ------------------
-        update(tid, hist->pc, actually_taken, hist->bpHistory,
+        cPred->update(tid, hist->pc, actually_taken, hist->bpHistory,
                true, hist->inst, corr_target.instAddr());
```

Comment:
Align this as well. The `true` should start on the same level as the `tid`
```suggestion
                     true, hist->inst, corr_target.instAddr());
```